docs/tutorials/control/air-quality-fleet.md

```diff
@@ -0,0 +1,952 @@
+
+   The setup page will indicate when the machine is successfully connected.
+
+1. If Antonia has more than one air sensing machine, add a new machine to her location and set it up in the same way.
+
+This is how you set up one machine.
+If you are following along for the RobotsRUs business from our example, create additional machines in each sub-location, that is, in the `Oregon Office` location and in the `New York Office` location.
+
+## Set up your hardware
+
+{{% alert title="Note" color="note" %}}
+If this were a real company and you were shipping air sensing machines to customers, you would have the customer plug in power to the machine wherever they are setting it up.
+Since you already installed `viam-server`, once a customer connects the machine to power and sets up wifi, the machine will automatically re-connect to the Viam app and pull any configuration updates.
+{{% /alert %}}
+
+For each sensing machine:
+
+1. Connect the PM sensor to a USB port on the machine's SBC.
+
+1. Position your sensing machines in strategic locations, and connect them to power.
+   Here are some ideas for where to place sensing machines:
+
+   - At home:
+     - In an outdoor location protected from weather, such as under the eaves of your home
+     - In the kitchen, where cooking can produce pollutants
+     - Anywhere you spend lots of time indoors and want to measure exposure to pollutants
+   - At work:
+     - At your desk to check your exposure throughout the day
+     - Near a door or window to see whether pollutants are leaking in
+
+## Configure your air quality sensors
+
+You need to [configure](/build/configure/) your hardware so that each of your machines can communicate with its attached air quality [sensor](/components/sensor/).
+
+No matter how many sensing machines you use, you can configure them efficiently by using a reusable configuration block called a _{{< glossary_tooltip term_id="fragment" text="fragment" >}}_.
+Fragments are a way to share and manage identical machine configurations across multiple machines.
+Instead of going through all the configuration steps for each machine, you'll start by configuring just one machine and create a fragment based on that machine's configuration.
+Then, you'll add the fragment to each of your machines.
+With all your machines configured using the same fragment, if you need to update the config in the future, you can just update the fragment and all machines will automatically get the update.
+
+{{< alert title="Note" color="note" >}}
+If this was a real company, adding the fragment to each individual machine would quickly become tiring.
+We're showing you how to do this manually as a learning device.
+Once you understand how to configure machines and use fragments, you can use [Provisioning](/fleet/provision/) to automatically set up your devices.
+{{< /alert >}}
+
+### Configure your first machine
+
+#### Configure the sensor
```

Comment:
```suggestion
## Configure your first machine

### Configure the sensor
```

Reply:
This is all within the H2 heading "Configure your air quality sensors," so I don't think these should be bumped out? Should I change the H2 heading to "Configure your fleet" instead of "Configure your air quality sensors"?

Reply:
No let's leave as is